packages/language-server/test/fixture/enhancedDiagnostics.astro

```diff
@@ -1,6 +1 @@
----
-import { getEntryBySlug } from 'astro:content';
-getEntryBySlug;
----
```

Comment:
This test no longer work since we now generate types in the tests. That's okay because the feature being tested here is still tested, just not.. double tested